features/step_definitions/steps.rb

```diff
@@ -0,0 +1,27 @@
+Given("I am on the landing page") do
+  visit 'welcome/index'
+end
+  
+  Given("I click on the {string} button") do |string|
+    click_on(string)
+  end
+  
+  Then("I fill in {string} with my {string}") do |y, x|
+    fill_in y, with: x
+  end
+  
+  Then("I will in {string} with my {string}") do |string, string2|
```

Comment:
This looks almost the same as the step definition above (see line 9). Avoid duplications like this.